Change "copy" icon | Issue #28700

jayantpranjal0 opened the pull request:
Changes the files copy_to_clipboard_svg.hbs to have the new icon from https://feathericons.com/?query=copy as required by the issue

Fixes: #28700

<!-- If the PR makes UI changes, always include one or more still screenshots to demonstrate your changes. If it seems helpful, add a screen capture of the new functionality as well.

Tooling tips: https://zulip.readthedocs.io/en/latest/tutorials/screenshot-and-gif-software.html
-->

**Screenshots and screen captures:**
Before:
![image](https://github.com/zulip/zulip/assets/96689162/8e770a7a-0397-4b93-8bf0-42c21d90189f)
After
<img width="950" alt="image" src="https://github.com/zulip/zulip/assets/96689162/4df95391-6f76-494a-83d7-e629c19ea851">

<img width="655" alt="image" src="https://github.com/zulip/zulip/assets/96689162/cbff375f-dd5e-44b6-8f5a-339ae72e2312">
<img width="773" alt="image" src="https://github.com/zulip/zulip/assets/96689162/d4cb79a1-3712-49fd-8b00-5cdcd676f278">


<details>
<summary>Self-review checklist</summary>

<!-- Prior to submitting a PR, follow our step-by-step guide to review your own code:
https://zulip.readthedocs.io/en/latest/contributing/code-reviewing.html#how-to-review-code -->

<!-- Once you create the PR, check off all the steps below that you have completed.
If any of these steps are not relevant or you have not completed, leave them unchecked.-->

- [x] [Self-reviewed](https://zulip.readthedocs.io/en/latest/contributing/code-reviewing.html#how-to-review-code) the changes for clarity and maintainability
      (variable names, code reuse, readability, etc.).

Completed manual review and testing of the following:

- [x] Visual appearance of the changes.
- [x] Responsiveness and internationalization.
- [x] Strings and tooltips.
- [x] End-to-end functionality of buttons, interactions and flows.
- [x] Corner cases, error conditions, and easily imagined bugs.
</details>

alya commented:
Please include screenshots of everywhere in the app that we have a "copy" icon, not just code blocks.

jayantpranjal0 commented:
@alya I have added the screenshots of everywhere where we have a "copy" icon. Please review the PR.

jayantpranjal0 commented:
@alya I have added the screenshots of everywhere where we have a "copy" icon. Please review the PR.

alya commented:
As you can see, there are major problems with the appearance of the icon, which it would have been helpful to point out.

It's my bad for not linking the Figma frame that has the icon. Can you try exporting it from there and seeing if that fixes the problem?

jayantpranjal0 commented:
> As you can see, there are major problems with the appearance of the icon, which it would have been helpful to point out.
> 
> It's my bad for not linking the Figma frame that has the icon. I've added a link to the issue. Can you try exporting it from there and seeing if that fixes the problem?

@alya I have changed changed the copy icon design according to the figma design link attached. Please review the PR.

alya commented:
Thanks! It looks in your screenshots like you've changed colors, and not just the icon? Was this requested somewhere?

Your PR also needs to pass test in order to be ready for review.

timabbott commented:
Closing in favor of https://github.com/zulip/zulip/pull/29455 for this issue.
